.github/workflows/pull-request.yml

```
@@ -126,7 +126,7 @@ jobs:
     name: Checks if sensitive files have been changed without authorization
     needs: [Code-Quality-Checks]
     runs-on: ubuntu-latest
-    steps:
+    steps:   
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)